src/methods/update.ts

```diff
@@ -0,0 +1,30 @@
+import { Document, Schema, OpaqueIndex, OpaqueDocumentStore, Lyra } from "../types.js";
+import { insert, insertMultiple } from "./insert.js";
+import { remove, removeMultiple } from "./remove.js";
+
+export async function update<S extends Schema, I extends OpaqueIndex, D extends OpaqueDocumentStore>(
+  lyra: Lyra<S, I, D>,
+  id: string,
+  doc: Document,
+  language?: string,
+  skipHooks?: boolean,
+): Promise<string> {
+  await remove(lyra, id, language, skipHooks);
```

Comment:
I think we should also add `beforeUpdate` and `afterUpdate` hooks. I know this is a shorthand for remove + insert, but you might want to run certain operations _specifically_ when updating

Reply:
Good point. Will update this.

Reply:
Done. Will be there in the next push.